### DISCUSSION TITLE
Missing log output for certain steps in Composite Action

### DISCUSSION THREAD

**Discussion by suyanhanx** [community/community#72079] [Actions]
### Select Topic Area

Product Feedback

### Body

Hello, GitHub community members!

I recently encountered an issue while using the Composite Action feature in GitHub Actions, and I'm seeking help and advice here.

## Issue Description

I created a Composite Action that includes multiple steps and operations. 
When using this Composite Action, I noticed that some of the steps do not show as seperate steps in the log output. 
This has caused confusion and obstacles while working with the Composite Action.

## Reproducible Steps

1. Use the Composite Action in a workflow.
2. Observe the log output, especially for certain steps.

## Expected Result

I expect to see the all steps of the action in the Composite Action.

## Actual Result

However, I found that there are some steps missing in the logs. Their logs were merged into the same step.

Is this on purpose? Is there a way to explicitly out separate steps.

I have already tried the following methods, but the issue still persists:

- Ensuring correct step configurations without syntax errors.
- Checking that the steps in the Composite Action definition are executed correctly.
- Reviewing similar issues and solutions from other users but haven't found a suitable answer.

I would appreciate any guidance, suggestions, or insights to resolve this issue. If anyone has encountered a similar situation or knows how to fix this problem, I would be grateful for your input.

Thank you for your assistance!

---
Additional Information:
- I'm using composite action in this project: https://github.com/apache/incubator-opendal/
- How we are using composite action: https://github.com/apache/incubator-opendal/blob/main/.github/workflows/behavior_test.yml
- An job matrix example: https://github.com/apache/incubator-opendal/actions/runs/6624360110?pr=3370
- An action job log example whose steps are missing in log: https://github.com/apache/incubator-opendal/actions/runs/6624360110/job/17993185660?pr=3370
- And its config file: https://github.com/apache/incubator-opendal/blob/main/.github/services/azblob/azurite_azblob/action.yml

**Comment by mickeygousset** [comment]
So what "steps" are you seeing as missing?

if I go into the Test Core step of the azurite_azblob job, I see all the different steps that make up the composite action.

https://github.com/apache/incubator-opendal/actions/runs/6624360110/job/17993185660?pr=3370#step:5:1

**Reply by suyanhanx** [reply]
Sorry for the misunderstanding. I also realized that all the logs are there, it's just that the steps for this action aren't showing up. That's the point I was trying to ask. I've updated the content.

**Comment by ssbarnea** [comment]
Apparently there is no `stderr` output displayed on composite actions and that is not documented.
